app/models/review.rb

```diff
@@ -4,4 +4,19 @@ class Review < ApplicationRecord
   belongs_to :user
   belongs_to :trail
 
+  has_many :likes
+  has_many :users, through: :likes
+
+
+    def votes_count
+      vote_count = 0
+      likes.each do |like|
+        if like.vote == "like"
+         vote_count += 1
+       elsif like.vote == "dislike"
+          vote_count -= 1
+        end
+      end
+      return vote_count
+    end
```

Comment:
A little nit picky, but this should probably be called `vote_count` instead of `votes_count`

Reply:
Agreed; changed